graphqlapi/src/main/java/io/stargate/graphql/schema/DmlSchemaBuilder.java

```diff
@@ -576,8 +585,11 @@ public GraphQLObjectType buildType(Table table) {
         log.error(String.format("Input type for %s could not be created", column.name()), e);
       }
     }
+    GraphQLInputObjectType filterInputType =
+        filterInputType(
+            typeNameBase + "FilterInput", new GraphQLTypeReference(typeNameBase + "Input"));
```

Comment:
@jorgebay We need something like this for collections too. Currently they're not included in the filter input object:
```
CQL:
create table foo(l frozen<list<int>> primary key, b int);

GraphQL:
input FooFilterInput {
  b: IntFilterInput
}
```
I'll address it while adding SAI support, since it introduces new comparison operators for collections.

Reply:
Good catch.